### DISCUSSION TITLE
My site is not indexed by google

### DISCUSSION THREAD

**Discussion by paolojulian** [vercel/community#3534] [Help] [answered]
### Summary

I am a newbie in SEO and web crawlers,
It seems like Vercel and Next.js is not seen in google searches.
There are no robot headers and stuff.

https://paolojulian.dev/blogs

### Example

https://paolojulian.dev/blogs

### Steps to Reproduce

site:https://paolojulian.dev/blogs showed no results

**Comment by leerob** [comment, marked as answer]
Everything seems okay with your site. I'd recommend learning more about SEO here: https://nextjs.org/learn/seo

And submitting your site to Google Search Console 👍
